lib/util.js

```diff
@@ -752,9 +784,11 @@ function formatValue(ctx, value, recurseTimes) {
       // Fast path for ArrayBuffer and SharedArrayBuffer.
       // Can't do the same for DataView because it has a non-primitive
       // .buffer property that we need to recurse for.
-      let prefix = getPrefix(constructor, tag);
+      const arrayType = isArrayBuffer(value) ? 'ArrayBuffer ' :
+        'SharedArrayBuffer ';
```

Comment:
Due to the changes the types have to be `'ArrayBuffer'` and `'SharedArrayBuffer'` (without the whitespace at the end).